core/modules/taxonomy/tests/taxonomy.test

```diff
@@ -59,14 +59,25 @@ class TaxonomyVocabularyFunctionalTest extends TaxonomyWebTestCase {
     // Visit the main taxonomy administration page.
     $this->backdropGet('admin/structure/taxonomy');
 
+    // Create a new vocabulary and then add terms.
+    $this->clickLink(t('Add vocabulary'));
+    $edit = array();
+    $machine_name = backdrop_strtolower($this->randomName());
+    $edit['name'] = $this->randomName();
+    $edit['description'] = $this->randomName();
+    $edit['machine_name'] = $machine_name;
+    $this->backdropPost(NULL, $edit, t('Save and add terms'));
+    $this->assertRaw(t('Created new vocabulary %name.', array('%name' => $edit['name'])));
```

Comment:
Let's make this match the next similar section of code:

```suggestion
    $this->assertRaw(t('Created new vocabulary %name.', array('%name' => $edit['name'])), 'Vocabulary created successfully.');
```

Should we also be checking that the redirect to the "add term" form works here? Checking for the page title and/or the current path should suffice I think.